modules/exploits/linux/http/hadoop_unauth_exec.rb

```diff
@@ -14,8 +14,8 @@ def initialize(info = {})
     super(update_info(info,
       'Name'           => 'Hadoop YARN ResourceManager Unauthenticated Command Execution',
       'Description'    => %q{
-          This module uses built-in functionality to execute arbitrary commands on an unsecured Hadoop server which is not configured for strong
-          authentication, via Hadoop's standard ResourceManager REST API.
+          This module uses built-in functionality to execute arbitrary commands on Hadoop servers which are not properly configured to be exposed to
+          the network, via Hadoop's standard ResourceManager REST API.
```

Comment:
I think the old explanation makes sense. This is about authentication mechanisms, not whether or not the server is exposed to the network or how much of its functionality is exposed. Ultimately the core issue here appears to be that you can access the ResourceManager unauthenticated. If that is exposed publicly but is protected by Kerberos authentication then your new suggested statement would be a violation of this rule, when in reality the issue described here no longer applies.